docker-compose.yml

```diff
@@ -54,6 +54,7 @@ services:
     image: openjdk:8-jre
     links:
       - postgres:database.service.rasterfoundry.internal
+      - memcached:tile-cache.service.rasterfoundry.internal
```

Comment:
Unrelated, but I think that the arguments to `memcached` should be a little bit more conservative:

```diff
diff --git a/docker-compose.yml b/docker-compose.yml
index 5821d03d..59be017c 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -10,7 +10,7 @@ services:
 
   memcached:
     image: memcached:1.4-alpine
-    command: -m 4096 -c 8192 -I 5242880b
+    command: -m 2048 -c 1024 -I 3145728b
 
   nginx-api:
     image: raster-foundry-nginx-api
```